Mathlib/RingTheory/TwoSidedIdeal/Commute.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2025 Antoine Chambert-Loir, María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir
+-/
+import Mathlib.Data.Nat.Choose.Sum
+import Mathlib.RingTheory.TwoSidedIdeal.BigOperators
+
+/-! # Powers of commuting elements in two sided ideals -/
```

Comment:
Do you still need this file, or is it a leftover from the PR on linear topologies where you didn't actually need it?